model/Database.php

```diff
@@ -345,5 +345,20 @@ function importUser($user_id) {
 
         $statement->execute([$user_id, $year]);
     }
+    /**
+     * Get user by it's email
+     * @param int $user_email users email
+     * @return a row of information for user
```

Comment:
@return requires datatype (I believe it would be an array for this?)

Reply:
Updated the datatype for @return